integration/tests/shellcheck_rendered_hooks.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#shellcheck disable=SC2034
+test_name="shellcheck hooks"
+test_deploy_inspec_profiles=()
+test_skip_diagnostics=true
+
+do_setup() {
+    do_setup_default
+
+    # We are defaulting to a umask of 077 to test
+    # installations on systems that are super locked down.
```

Comment:
Suggest:
```suggestion
    # installations on systems that are completely locked down.
```

Reply:
Unfortunately, I've copy-pasted that bit from the other places. I propose we turn it into a helper and fix the comment there.